apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.utils.ts

```diff
@@ -123,18 +123,20 @@ export const generateUpdateColumnPayload = (
   const isOriginallyPrimaryKey = primaryKeyColumns.includes(originalColumn.name)
 
   // Only append the properties which are getting updated
+  const name = field.name.trim()
   const type = field.isArray ? `${field.format}[]` : field.format
-  const comment = (field.comment?.length ?? '') === 0 ? null : field.comment
+  const comment = ((field.comment?.length ?? '') === 0 ? null : field.comment)?.trim()
+  const check = field.check?.trim()
 
   const payload: Partial<UpdateColumnPayload> = {}
-  if (!isEqual(originalColumn.name, field.name)) {
-    payload.name = field.name
+  if (!isEqual(originalColumn.name, name)) {
+    payload.name = name
   }
   if (!isEqual(originalColumn.comment, comment)) {
```

Comment:
Renaming the comment should be fine unless they're using the comments for something else like postgraphile etc and they need some crazy format or something... so maybe worth adding a `.trim()` here as well?